root/etc/cont-init.d/20-outrigger-private-key

```diff
@@ -3,10 +3,9 @@
 KEY_BASE=/root/.ssh
 KEY_FILE=$KEY_BASE/outrigger.key
 
-if [ -e $KEY_FILE ]; then
-
+if [ -e $KEY_FILE ] && [ ! -d $KEY_FILE ]
```

Comment:
If you mount a non existent file on your host with a volume mount an empty directory shows up on both sides.